#! /bin/sh /usr/share/dpatch/dpatch-run
## 0023_remove_useless_linking.dpatch by  <fabbe@paniq.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Avoid linking unecessarily against libkrb5support.

@DPATCH@
diff -urNad trunk~/cmulocal/sasl2.m4 trunk/cmulocal/sasl2.m4
--- trunk~/cmulocal/sasl2.m4	2006-06-04 12:26:17.000000000 +0300
+++ trunk/cmulocal/sasl2.m4	2009-05-13 16:51:19.000000000 +0300
@@ -110,9 +110,6 @@
   fi
 
   if test "$gss_impl" = "auto" -o "$gss_impl" = "mit"; then
-    # check for libkrb5support first
-    AC_CHECK_LIB(krb5support,krb5int_getspecific,K5SUP=-lkrb5support K5SUPSTATIC=$gssapi_dir/libkrb5support.a,,${LIB_SOCKET})
-
     gss_failed=0
     AC_CHECK_LIB(gssapi_krb5,gss_unwrap,gss_impl="mit",gss_failed=1,
                  ${GSSAPIBASE_LIBS} -lgssapi_krb5 -lkrb5 -lk5crypto -lcom_err ${K5SUP} ${LIB_SOCKET})
